packages/smooth_app/lib/pages/product/summary_card.dart

```diff
@@ -406,6 +423,11 @@ class _SummaryCardState extends State<SummaryCard> {
                     ),
                   ),
                 );
+                // Reload the product questions, they might have been answered.
+                // Or the backend may have new ones.
+                await loadProductQuestions();
+                // Rebuild as the questions may have changed.
+                setState(() {});
```

Comment:
Shouldn't you reload the product as well: the answered questions are supposed to alter the product, aren't they?

Reply:
Not really, the product won't get instantly updated, it should take easily 5-10 minutes. One of the feature request is to send a notification once the product is updated. How we could handle real time update is the client could listen to a notification from the BE, once it gets a signal it refreshes the product (if the user is on the product page), or sends a notification to users's phone.